sql-statements/sql-statement-admin-show-ddl.md

```diff
@@ -11,7 +11,12 @@ The `ADMIN SHOW DDL [JOBS|JOB QUERIES]` statement shows information about runnin
 
 ```ebnf+diagram
 AdminStmt ::=
-    'ADMIN' ( 'SHOW' ( 'DDL' ( 'JOBS' Int64Num? WhereClauseOptional | 'JOB' 'QUERIES' NumList | 'JOB' 'QUERIES' 'LIMIT' m 'OFFSET' n )? | TableName 'NEXT_ROW_ID' | 'SLOW' AdminShowSlow ) | 'CHECK' ( 'TABLE' TableNameList | 'INDEX' TableName Identifier ( HandleRange ( ',' HandleRange )* )? ) | 'RECOVER' 'INDEX' TableName Identifier | 'CLEANUP' ( 'INDEX' TableName Identifier | 'TABLE' 'LOCK' TableNameList ) | 'CHECKSUM' 'TABLE' TableNameList | 'CANCEL' 'DDL' 'JOBS' NumList | 'RELOAD' ( 'EXPR_PUSHDOWN_BLACKLIST' | 'OPT_RULE_BLACKLIST' | 'BINDINGS' ) | 'PLUGINS' ( 'ENABLE' | 'DISABLE' ) PluginNameList | 'REPAIR' 'TABLE' TableName CreateTableStmt | ( 'FLUSH' | 'CAPTURE' | 'EVOLVE' ) 'BINDINGS' )
+    'ADMIN' 'SHOW' 'DDL'
```

Comment:
Do we need to specify the different meanings of three usages
```
"admin show ddl job queries limit m" -> "ADMIN SHOW DDL JOB QUERIES LIMIT 0, m"
"admin show ddl job queries limit n, m" -> "ADMIN SHOW DDL JOB QUERIES LIMIT n, m"
"admin show ddl job queries limit m offset n" -> "ADMIN SHOW DDL JOB QUERIES LIMIT n, m"
```

Reply:
I don't think that's needed. Most people will already be familiar with this because of SELECT statements